app/views/wiki/show.html.erb

```diff
@@ -64,11 +64,15 @@
     <% end %>
 
     <div class="tab-pane active" id="tab-overview">
-      <div <% unless @node.has_tag('style:wide') %>style="max-width:800px;"<% end %> id="content" class="pl-content wiki">
-        <%= raw auto_link(insert_extras(@revision.render_body), sanitize: false) %>
-      </div>
-      <% if params[:raw] == 'true' %>
-        <div style="display:none;" id="content-raw-markdown" class="pl-content wiki"><%= raw insert_extras(@revision.body_raw) %></div>
+      <% if params[:comments] == 'true' %>
```

Comment:
Ah sure, but then what I will do is render the `show` action of `wiki_controller` from that newly created action in `comment_controller.rb` while passing the `id` of the wiki ?. 

And in the `show` action of `wiki_controller` I will check the controller and action it was rendered from and, if it was rendered from the action we just created within the `comment_controller`, I will render the comment tab ?